enterprise_access/apps/content_assignments/management/commands/automatically_nudge_assignments.py

```diff
@@ -0,0 +1,135 @@
+        )
+        parser.add_argument(
+            '--days_before_course_start_date',
+            action='store_true',
+            dest='days_before_course_start_date',
+            default=30,
+            help='The amount of days before the course start date to send a nudge email through braze',
+        )
+
+    @staticmethod
+    def to_datetime(value):
+        """
+        Return a datetime object of `value` if it is a str.
+        """
+        if isinstance(value, str):
+            return datetime.datetime.strptime(
+                value,
+                "%Y-%m-%dT%H:%M:%SZ"
+            ).replace(
+                tzinfo=datetime.timezone.utc
+            )
+
+        return value
+
+    def handle(self, *args, **options):
+        dry_run = options['dry_run']
+        days_before_course_start_date = options['days_before_course_start_date']
+
+        for assignment_configuration in AssignmentConfiguration.objects.filter(active=True):
+            subsidy_access_policy = assignment_configuration.subsidy_access_policy
+            enterprise_catalog_uuid = subsidy_access_policy.catalog_uuid
+
+            message = (
+                '[AUTOMATICALLY_REMIND_ACCEPTED_ASSIGNMENTS_1] Assignment Configuration. UUID: [%s], '
+                'Policy: [%s], Catalog: [%s], Enterprise: [%s], dry_run [%s]',
+            )
+            logger.info(
+                message,
+                assignment_configuration.uuid,
+                subsidy_access_policy.uuid,
+                enterprise_catalog_uuid,
+                assignment_configuration.enterprise_customer_uuid,
+                dry_run,
+            )
+
+            accepted_assignments = assignment_configuration.assignments.filter(
+                state=LearnerContentAssignmentStateChoices.ACCEPTED
+            )
+
+            paginator = Paginator(accepted_assignments, 100)
+            for page_number in paginator.page_range:
+                assignments = paginator.page(page_number)
+
+                content_metadata_for_assignments = get_content_metadata_for_assignments(
+                    enterprise_catalog_uuid,
+                    assignments
+                )
+
+                for assignment in assignments:
+                    content_metadata = content_metadata_for_assignments.get(assignment.content_key, {})
+                    start_date = content_metadata.get('normalized_metadata', {}).get('start_date')
+                    course_type = content_metadata.get('course_type')
+
+                    is_executive_education_course_type = course_type in (
+                        'executive-education-2u', 'executive-education')
+
+                    # Determine if the date from today + days_before_course_state_date is
+                    # equal to the date of the start date
+                    # If they are equal, then send the nudge email, otherwise continue
+                    datetime_start_date = self.to_datetime(start_date)
```

Comment:
There's an existing `parse_datetime_string` in here, it's designed for reading several different possible date formats from content metadata records.

Reply:
...if you want to be a good citizen, you could add an optional kwarg to that function like `set_to_utc` that, if true, will return a datetime object with the `.replace(tzinfo=datetime.timezone.utc)` tacked-on, like you have above.